dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_34__Add_programstagedataelement_skipanalytics.sql

```diff
@@ -0,0 +1,5 @@
+
+alter table "programstagedataelement" add column if not exists "skipanalytics" boolean;
+update "programstagedataelement" set "skipanalytics" = false where "skipanalytics" is null;
+alter table "programstagedataelement" alter column "skipanalytics" set not null;
```

Comment:
Is there a reason to not do this in 1 step?

Reply:
Reason is that we cannot add a not-null column to a table with existing rows, as it would fail. We could set a default value directly for the column, but then the default is suddenly defined in the database and not in application code/entity class, which is a new approach and inconsistent with what we typically do. Again, we need a standard approach for default values.